wix/serverTimestamp.config

```diff
@@ -0,0 +1,5 @@
+; comment with semicolon
+; blank line are ingnored
+; script use only first column
+http://timestamp.verisign.com/scripts/timstamp.dll

```

Comment:
Is that the right UR? missing an `e`?

Reply:
fixed

Reply:
@gdams  have you tried both with sha1 and sha256 ?